build.wake

```diff
@@ -133,7 +133,7 @@ def infoModeToString infoMode = match infoMode
   FirrtlInfoAppend = "append"
 
 tuple FirrtlCompileOutputs =
-  TargetOutputs_: List Path
+  global TargetOutputs_: List Path
```

Comment:
I'm not understanding why this is necessary. I can reproduce locally the error `Type signature 'FirrtlCompileOutputs' refers to a non-existent type [[REDACTED FILE PATH]].wake:3[28-47]`, but it's not obvious to me why this is a problem. That particular line happens to be in the definition of another `tuple` that refers to this `FirrtlCompileOutputs ` tuple.

My best guess is that in Wake 0.18 you went with a rule of making a `tuple` type globally-accessible if and only if at least one of its members is `global`, and that the choice of making this particular member global was arbitrary. If this is the case, can you please make sure this is documented in Wake 0.18? This is not obvious behavior.

Reply:
This is documented in https://github.com/sifive/wake/blob/master/share/doc/wake/tour/packages.adoc

> The expression export tuple Friend in the example triggers both the export of the type Friend and the definition Friend, used to construct a Friend and match it in patterns. The subsequent export Name triggers the export of the type Friend and definitions getFriendName, setFriendName, and editFriendName.

... FYI, you do not need to do this with `export`.

You can post-hoc do a: `from my_package export type MyTuple` to export the type without a tuple member. This is does not work with the legacy-global support

Reply:
Also based on an off-GitHub conversation @terpstra and I had, we think that we can replace this change with a change that adds a temporary new, `global` member that is of type `Unit`, which preserves the visibility of the existing members and does not affect functionality because that member would be useless.